### DISCUSSION TITLE
angular/how-to-use-font-awesome-icons-in-angular-applications/

### DISCUSSION THREAD

**Discussion by giscus** [arungudelli/angularjswiki#197] [Announcements]
# angular/how-to-use-font-awesome-icons-in-angular-applications/

Step by step tutotial to use Font Awesome icons in Angular applications

https://www.angularjswiki.com/angular/how-to-use-font-awesome-icons-in-angular-applications/

**Comment by DoubyCz** [comment]
Using of global library that way is deprecated now, so you shoud use something like this instead: https://stackoverflow.com/a/58464715/806456

**Reply by DoubyCz** [reply]
sorry, wrong link ... this is the right one https://stackoverflow.com/a/69403267/806456

**Reply by arungudelli** [reply]
Thanks updated the article
